gradle-failure-reports/src/main/java/com/palantir/gradle/failurereports/FailureReportsExtension.java

```diff
@@ -34,5 +38,6 @@ public FailureReportsExtension() {
                 .convention(getProjectLayout().getBuildDirectory().file("failure-reports/build-TEST.xml"));
         getFailureReportCompileOutputFile()
                 .convention(getProjectLayout().getBuildDirectory().file("failure-reports/build-compile-TEST.xml"));
+        getIgnoredTasks().add(Test.class);
```

Comment:
Is there a way to throw if the consumers are trying to do `getIgnoredTasks().set(...)` ?